### DISCUSSION TITLE
Plugin for custom reporting when running tests on remote desktop.

### DISCUSSION THREAD

**Discussion by noSpamsPlease** [cypress-io/cypress#28178] [Plugins] [answered]
What plugins can be configured to fetch custom reports when running Cypress tests on remote hosted machines

**Comment by grandEL-dsouza** [comment, marked as answer]
You can use the Allure reporting plugin to get custom reports. 

For more details about the Allure plug in please refer to this link:
https://www.npmjs.com/package/@shelex/cypress-allure-plugin

Since you are running it on a remotely hosted machine please note that you would have to configure the Allure report in config file and the downloads property should not be within the package_config_options property. Instead it should directly come under run_settings . So your remoteDesktop.json file should look like this same code snippet. 

`{
...
"run_settings": {
  ...
  "downloads": ["./results", "./allure-results"],
  "package_config_options": {
      "cypress-cucumber-preprocessor": {
        "nonGlobalStepDefinitions": true
      }
  },
  ...
}
...
}`


I hope this helps. 

Thanks.
